scripts/engine/world.py

```diff
@@ -965,8 +969,15 @@ def recompute_fov(entity: EntityID) -> bool:
 
         # compute the fov
         if fov and pos:
-            tcod.map_compute_fov(fov.map, pos.x, pos.y, sight_range, fov.light_walls, fov.algorithm)
-
+            tranparency = create_fov_map()
+            maps = []
+            for coordinate in pos.get_coordinates():
+                map = tcod.map.compute_fov(tranparency, coordinate, sight_range, fov.light_walls, fov.algorithm)
+                maps.append(map)
+
+            fov.map = maps[0]
+            for m in maps:
+                fov.map |= m
```

Comment:
What is this doing? Is `|=` a bit computation?

Reply:
Yes. For each FOV map we calculated from each tile we combine them together with an OR operation. This makes it so that if a tile was visible in any of the maps it will be visible in the final one.

Reply:
You're testing me on my memory of bitwise operations. ;)

Reply:
Haha, they sometimes are really useful!